content/articles/spf-record.md

```diff
@@ -16,12 +16,18 @@ categories:
 ---
 
 <note>
-Although the SPF record type is deprecated, it's still supported by DNSimple name servers. We will discontinue serving SPF records in the future, so we will create a TXT record with the same content for you.
+  Effective August 11th 2025, legacy SPF records will no longer be supported. For more information, see our [announcement](blog-post-link).
+
+  This documentation has been updated to include the current recommended practices for SPF record management.
 </note>
 
 ## What's an SPF record?
 
-An SPF record is a Sender Policy Framework record. It's used to indicate to mail exchanges which hosts are authorized to send mail for a domain.
+An SPF record is a Sender Policy Framework record. It's used to indicate to mail exchanges which hosts are authorized to send mail for a domain. SPF records are specially formatted TXT records.
+
+## Creating an SPF record
+
+To create an SPF record follow the instructions for [creating a record](/articles/record-editor/#create-a-record), selecting TXT as the record type. The record's content follows SPF-specific formatting rules as described below.
```

Comment:
@itsalyse This is the latest update that I created. We need your help making this sound better 😂